packages/ai-core/src/internal.ts

```diff
@@ -0,0 +1 @@
+export * from './utils/index.js';
```

Comment:
[q] If I remember correctly, everything had to be exported from internal.ts? Or did we change the script?

Reply:
but in the other direction: we will allow to export whatever you want. 😊 @shibeshduw if you do that can you make sure to remove my ugly workarounds 😬